charts/pega/README.md

```diff
@@ -823,4 +823,46 @@ certificates:
     badssl.cer: |
       "-----BEGIN CERTIFICATE-----\n<<certificate content>>\n-----END CERTIFICATE-----\n"
 
-```
\ No newline at end of file
+```
+## Clustering Service Deployment (Draft)
+
+The values.yaml section for `hazelcast` is used for configuring a Client Server form of deployment, Pega nodes act as Client for the hazelcast server nodes in this form of deployment. It is supported for Pega Platform 8.6 or later. 
+
+Specifying your clustering service image for `hazelcast.image` and setting  `hazelcast.enabled` as `true` deploys Pega Platform and Hazelcast cluster in a client server arrangement. In latter arrangement, nodes running hazelcast startup independently of Pega Nodes and create a cluster
+having member count as specified in `hazelcast.replicas` parameter. Pega nodes connects to this cluster as client.
+
+Provide information on how discovery takes place, Ref: https://github.com/hazelcast/hazelcast-kubernetes
+
+**Note:** If you are deploying Pega Platform below release 8.6, you need to set `hazelcast.enabled` as `false`, otherwise the installation will fail. 
+
+Setting `hazelcast.enabled` as `false` deploys Pega and hazelcast in an embedded arrangement, in which hazelcast and Pega Platform run on the same node. Although from Pega 8.6 onwards it is highly recommended to use Client Server form of deployment.
```

Comment:
The statement can be - C/s is default and recommended . Emebdded is only for backward compatibility. So, the message is uniform